R/router.R

```diff
@@ -160,11 +173,79 @@ create_router_callback <- function(root, routes) {
   }
 }
 
+#' Create router UI
+#'
+#' Creates router UI in Shiny applications.
+#'
+#' @param default Main route to which all invalid routes should redirect.
+#' @param ... All other routes defined with shiny.router::route function.
+#' @param page_404 Styling of page when wrong bookmark is open. See \link{page404}.
```

Comment:
What is the difference between an invalid route and a wrong bookmark?

Reply:
There is no difference - this was just copy-pasted from the previous implementation. I'll change to "invalid route" to make this more clear.